docs/7.x/milv.config.yaml

```diff
@@ -23,5 +23,30 @@ white-list-external:
   - "http://127.0.0.1:3000/debug/pprof"
   - "https://en.wikipedia.org/wiki/Shebang_(Unix"
   - "https://[route53_domain](#route53_domain"
+  - "http://10.5.5.28:32010"
+  - "http://10.5.5.29:32010"
+  - "https://10.5.5.28:32009/web"
+  - "https://<cluster-hostname>/portalapi/v1/oidc/callback"
+  - "https://<host>/portalapi/v1/saml/callback"
+  - "https://gravitational.com/gravity/docs/ver/6.x/config.md#example-google-oidc-connector"
+  - "https://gravitational.com/gravity/docs/ver/6.x/config.md#tls-key-pair"
+  - "https://gravitational.com/gravity/docs/ver/6.x/config.md#example-provisioning-a-cluster-admin-user"
+  - "https://gravitational.com/gravity/docs/ver/6.x/config.md#trusted-clusters-enterprise"
+  - "https://gravitational.com/gravity/docs/config.md#trusted-clusters-enterprise"
+  - "http://localhost:6600/hub/#installing-gravity-hub"
 black-list:
-files:
\ No newline at end of file
+  - "./changelog.md"
```

Comment:
I saw that changelog.md was blacklisted earlier, but I wasn't sure about that. https://gravitational.com/gravity/docs/changelog/ is a customer facing resource.  Is this blacklisting something else?